protocol/protocol.tex

```diff
@@ -2717,17 +2918,25 @@
 \end{enumerate}
 } %saplingonward
 
-\vspace{-2ex}
-\nnote{In \zcashd, all \Sapling keys and addresses are derived according to \cite{ZIP-32}.}
+\orchardonward{
+An \Orchard \spendingKey $\SpendingKey$ is used to derive a \authSigningKey $\AuthSignPrivate$,
+and a \fullViewingKey $(\AuthSignPublic, \NullifierKey, \CommitIvkRand)$. From the \fullViewingKey
+we can also derive an \incomingViewingKey $\InViewingKey$, an \outgoingViewingKey $\OutViewingKey$, and
+a set of \diversifiedPaymentAddresses $\DiversifiedPaymentAddress = (\Diversifier, \DiversifiedTransmitPublic)$,
+as described in \crossref{orchardkeycomponents}.
+} %orchardonward
 } %saplingonward
 
+\vspace{-2ex}
+\sapling{\nnote{In \zcashd, all \SaplingAndOrchard keys and addresses are derived according to \cite{ZIP-32}.}}
```

Comment:
We don't plan to use ZIP 32 to derive Orchard keys, because we don't plan to support non-hardened derivation for Orchard. (Even if we did want to use ZIP 32, we couldn't use it as-is; we would need to update it for Orchard's key components.)

Reply:
Turns out that what we actually needed to do was basically what I specified for Sprout, so I've updated ZIP 32 in #448 with Orchard support.